### PR TITLE
executor: fix admin test

### PR DESCRIPTION
**Pull request by ekexium**
Signed-off-by: ekexium <ekexium@gmail.com>

### What problem does this PR solve?

Issue Number: close #xxx

No related issue since it's intended to fix a few broken tests or bugs caused by a merge conflict in a development branch.

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by bb7133** [comment]
Hi @ekexium , please link to an issue for your PR; if it is not necessary(I guess think this is not the case for your PR), please give a reason for it in your PR description.

File: executor/admin_test.go [hunk 1]
```diff
@@ -1322,144 +1322,12 @@ func TestCheckFailReport(t *testing.T) {
 	}()
 }
 
-func (s *testSuiteJoinSerial) TestAdminCheckTableFailed(t *testing.T) {
```
Comment (bb7133) [inline executor/admin_test.go:1325]: Why removing this test case?

Comment (ekexium) [reply executor/admin_test.go:1325]: It's been moved to admin_serial_test.go. The deleted part is mistakenly reserved from a merge.

**Review by cfzjywxk — APPROVED** [review]

**Comment by cfzjywxk** [comment]
@MyonKeminta 
PTAL, we could merge it fast in dev branch.

**Review by sticnarf — APPROVED** [review]

**Pull request merged**